otp.js

```diff
@@ -62,10 +74,14 @@ const sendOTP = async (phoneNumber, otp) => {
     })
 }
 
-async function begin(phoneNumber) {
-    const otp = getOTP();
-    await cacheOTP(phoneNumber, otp);
-    await sendOTP(phoneNumber, otp);
+async function begin(phoneNumber, ipAddresses) {
```

Comment:
This still seems to use country for IP address. It also is called with three arguments in `check-number.js` line 32 but only takes two